vector-im/element-x-android
# Fix issues around read receipt

## Type of change


## Content


This PR fixes several issues around read receipt:
- make the RR list scrollable in the bottom sheet of RR
- render read receipt for state events
- render aggregated read receipt for grouped state events. Note that the RRs are not clickable here, the group has to be expanded first.
- add preview for expanded group of state event

## Motivation and context


Reliable read receipt feature

## Screenshots / GIFs



## Tests



- Enable RR rendered in advanced settings.
- Open a room with many people, and observe that Read Receipt are rendered correctly. Note: there is still an issue on Rust side: https://github.com/matrix-org/matrix-rust-sdk/issues/2889

## Tested devices

- OS version(s):

## Checklist
